appveyor.yml

```diff
@@ -0,0 +1,335 @@
+                        See https://system.data.sqlite.org/index.html/tktview/44a0955ea344a777ffdbcc077831e1adc8b77a36
+                        and https://github.com/nhibernate/nhibernate-core/issues/1362 #>
+                    # Please note the connection string is formated for putting the db file in $configDir.
+                    'connection.connection_string' = "Data Source=$configDir\NHibernate.db;DateTimeFormatString=yyyy-MM-dd HH:mm:ss.FFFFFFF;";
+                    'connection.driver_class' = 'NHibernate.Driver.SQLite20Driver';
+                    'dialect' = 'NHibernate.Dialect.SQLiteDialect'
+                };
+                'SqlServerCe' = @{
+                    # Please note the connection string is formated for putting the db file in $configDir.
+                    'connection.connection_string' = "Data Source=$configDir\NHibernate.sdf;";
+                    'connection.driver_class' = 'NHibernate.Driver.SqlServerCeDriver';
+                    'command_timeout' = '0';
+                    'dialect' = 'NHibernate.Dialect.MsSqlCe40Dialect'
+                };
+                'SqlServer2008' = @{'connection.connection_string' = 'Server=(local)\SQL2017;User ID=sa;Password=Password12!;initial catalog=nhibernate;'};
+                'SqlServer2012' = @{
+                    'connection.connection_string' = 'Server=(local)\SQL2017;User ID=sa;Password=Password12!;initial catalog=nhibernate;';
+                    'dialect' = 'NHibernate.Dialect.MsSql2012Dialect'
+                }
+            }
+
+            #Settings for current build
+            $settings = $allSettings.Get_Item($env:db)
+            if (!$settings) {
+                Write-Host "Unable to find $env:db settings" -foregroundcolor 'white' -backgroundcolor 'red'
+                exit 1
+            }
+
+            #Create settings file
+            $configFile = (Join-Path $configDir 'hibernate.cfg.xml')
+            New-Item $configDir -Type Directory
+            cp (Join-Path $env:APPVEYOR_BUILD_FOLDER 'build-common\teamcity-hibernate.cfg.xml') $configFile
+
+            #Patch settings file
+            $config = [xml] (Get-Content $configFile)
+            $allProps = $config.'hibernate-configuration'.'session-factory'.property
+            foreach($key in $settings.keys)
+            {
+                $value = $settings[$key]
+                $property = $allProps | where {$_.name -eq $key}
+                if (!$property) {
+                    Write-Host "Unable to find $key property" -foregroundcolor 'white' -backgroundcolor 'red'
+                    exit 1
+                }
+                $property.InnerText = $value
+            }
+            $config.Save($configFile)
+        }
+
+    - ps: >-
+        Invoke-Command -ScriptBlock {
+            switch ($env:db) {
+                'Firebird' {
+                    # Install Firebird
+                    # Firebird zip is cached, check if we have it, otherwise download it
```

Comment:
There is choco package for Firebird too. I have not tested it, and it currently has a failing status. This manual install is not complicated, so I have not estimated it was trying the choco package.